src/Cohesity.Powershell/Cohesity.PowerShell.psm1

```diff
@@ -12,4 +12,9 @@ Foreach($script in @($scripts))
     {
         Write-Error -Message "Failed to import function $($script.fullname): $_"
     }
+}
+# Post module installation initialize the cmdlet configuration
+$config = Get-CohesityCmdletConfig
+if($null -eq $config) {
+    $resp = Set-CohesityCmdletConfig -LogSeverity 3
```

Comment:
Try and make sure the end line char is removed automatically.

Reply:
Fixed